jekyll/_docs/language-ruby-on-rails.md

```diff
@@ -25,12 +25,24 @@ If you don't want to use the default, you can specify your version in `circle.ym
 ```
 machine:
   ruby:
-    version: rbx-2.2.6
+    version: 2.4.0
 ```
 
-Our [test environment doc]({{site.baseurl}}/language-ruby-on-rails/)
-covers more details about language versions and tools; it also explains how Circle
-works with testing tools that require a browser.
+In the current version of CircleCI, we use `rvm` to manage and install different versions of Ruby. This has a couple of side-effects that users should be aware of.
```

Comment:
Suggested:

"In the current version of CircleCI, we use `rvm` to manage and install different versions of Ruby. This has a couple side-effects:"